core/src/main/java/io/fundrequest/core/request/claim/event/RequestClaimPendingEvent.java

```diff
@@ -0,0 +1,16 @@
+package io.fundrequest.core.request.claim.event;
+
+import io.fundrequest.core.request.claim.domain.RequestClaim;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+public class RequestClaimPendingEvent {
```

Comment:
It is better to let the name of the event reflect on the action that occurred (e.g. ClaimRequestedEvent) than on the state of the request after the action